### PR TITLE
fixed(#254):Notification message - Confirmation 

### PR DESCRIPTION
**Pull request by Pikachu-345**
# Done the following tasks:

-Made a new type "Feature Cards"
-Added the "confirmation message" component
-Modified the related mdx file accordingly

Please review the changes and let me know if you want changes..

### CODE REVIEW

**Comment by Pikachu-345** [comment]
@hari please check this pr for #254

File: animata/feature-cards/confirmation-message.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  /**
+   * Class name for the container element.
+   */
+  containerClassName?: string;
+}
+
+export default function ConfirmationMessage({
+  successMessage = "Process Successful",
+  labelName = "Animata",
+  labelMessage = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque tincidunt. Vestibulum eleifend nibh a est tristique sollicitudin.",
```
Comment (hari) [inline animata/feature-cards/confirmation-message.tsx:35]: Let's add a meaningful sentence here.

File: content/docs/feature-cards/confirmation-message.mdx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+title: Confirmation Message
+description: An animated component which shows success message with custom name and description.
+author: YourTwitterUsername
```
Comment (hari) [inline content/docs/feature-cards/confirmation-message.mdx:4]: Your Twitter username

**Comment by hari** [comment]
@Pikachu-345 Please update the following:
1. Improve colors
2. Limit max width
3. Do not animate the height of the container because the content is shifting. You can add a parent div to it and animate its height instead, make sure to add `relative overflow-hidden max-h-fit` to it.

The animation can also be done using vanilla CSS and JS, you don't necessarily need framer-motion. I will leave this up to you.

File: animata/feature-cards/confirmation-message.tsx (hunk 1 above)
Comment (Pikachu-345) [reply animata/feature-cards/confirmation-message.tsx:35]: suggestion are welcome.. for this "meaningful sentence" @hari

**Review by hari — APPROVED** [review]

**Comment by Pikachu-345** [comment]
so are changes any chnages required?
If not please merge the PR

**Pull request merged**